samples/sensor/ds18b20/README.rst

```diff
@@ -0,0 +1,44 @@
+.. _ds18b20:
+
+DS18B20: Temperature Sensor
+###########################
+
+Overview
+********
+This sample periodically reads temperature and humidity from the DS18B20
+Temperature Sensor and displays it on the console
+
+
+Requirements
+************
+
+This sample uses the DS18B20 sensor controlled using the 1-Wire interface.
+
+References
+**********
+
+- DS18B20: https://www.maximintegrated.com/en/app-notes/index.mvp/id/162
+
+Building and Running
+********************
+
+This project outputs sensor data to the console. It requires an DS18B20
+sensor.
+
+.. zephyr-app-commands::
+   :zephyr-app: samples/sensors/ds18b20
+   :board: stm32_min_dev
+   :goals: build
+   :compact:
+
+Sample Output
+=============
+
+.. code-block:: console
+
+  Temperature:25.3 C
```

Comment:
Almost.  The contents of the code-block should be indented to the ``c`` in ``code-block`` (otherwise the content is not considered part of the code-block).  Add one more space in front of each of the code-block contents lines (the ``.. code-block::`` directive itself is fine.)

Reply:
done